x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -90,12 +98,23 @@ const navigationTree: NavigationTreeDefinition = {
               children: [
                 {
                   link: 'apm:services',
+                  getIsActive: ({ pathNameSerialized, prepend }) => {
+                    return (
+                      pathNameSerialized.startsWith(prepend('/app/apm/services')) ||
```

Comment:
I was wondering if we could check this using a regex instead of including all routes here
something like `\/app\/apm\/.*service.*` would work for all the routes. Or do we prefer just to explicitly define the routes as we are doing?